packages/core/core/src/PackagerRunner.js

```diff
@@ -73,121 +84,84 @@ export default class PackagerRunner {
     let farm = nullthrows(this.farm);
     let {ref, dispose} = await farm.createSharedReference(bundleGraph);
 
-    let promises = [];
-    for (let bundle of bundleGraph.getBundles()) {
-      // skip inline bundles, they will be processed via the parent bundle
-      if (bundle.isInline) {
-        continue;
-      }
-
-      promises.push(
-        this.writeBundle(bundle, bundleGraph, ref).then(stats => {
-          bundle.stats = stats;
+    let bundleInfoMap = {};
+    // ? hashRef should maybe just be first 8 digits of id, but that would require that id is an md5 hash already
+    //  - only reason I'm not doing that right now is that it feels like we're doing a bunch of hashes of hashes
+    //  - not sure if that's a bad thing or not
```

Comment:
Thoughts?

Reply:
Implemented offline feedback in [cc6fd15](https://github.com/parcel-bundler/parcel/pull/4114/commits/cc6fd158443fa3c63e8be7bf8baf0f533a9831f4)